drivers/ma_reports/app/models/ma_reports/monthly_performance/report.rb

```diff
@@ -18,6 +18,12 @@ class Report < SimpleReports::ReportInstance
     has_many :enrollments
     has_many :projects
 
+    private def expires_in
+      return 30.seconds if Rails.env.development?
```

Comment:
Might want to increase this even for development since the data is pre-calculated and shouldn't be changing. (Maybe 5 days?)